### PR TITLE
ISPN-14300 Add a delta in a value

### PR DESCRIPTION
**Pull request by dpanshug**
https://issues.redhat.com/browse/ISPN-14300

### CODE REVIEW

File: src/app/Counters/AddDeltaCounter.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, ButtonVariant, Modal, Text, TextContent, TextInput } from '@patternfly/react-core';
+import { useAddDeltaCounter } from '@app/services/countersHook';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * Delete counter modal
+ */
+const AddDeltaCounter = (props: {
```
Comment (karesti) [inline src/app/Counters/AddDeltaCounter.tsx:9]: I would like a test for this, mocking the counter modal if possible. Unit test one I mean

File: src/app/Counters/AddDeltaCounter.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, ButtonVariant, Modal, Text, TextContent, TextInput } from '@patternfly/react-core';
+import { useAddDeltaCounter } from '@app/services/countersHook';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * Delete counter modal
```
Comment (karesti) [inline src/app/Counters/AddDeltaCounter.tsx:7]: Add delta

File: src/app/assets/languages/en.json [hunk 3]
```diff
@@ -121,7 +121,13 @@
       "counter-type": "Counter type",
       "no-counters-status": "No counters yet",
       "no-counters-body": "Click \"Create a counter\" and configure a counter. You can also create counters from the CLI or remote clients.",
-      "no-filtered-counter-body": "No counters match your filter criteria. Click \"Create a counter\" and configure a counter."
+      "no-filtered-counter-body": "No counters match your filter criteria. Click \"Create a counter\" and configure a counter.",
+      "delete-action": "Delete",
+      "add-delta-action": "Add delta",
```
Comment (karesti) [inline src/app/assets/languages/en.json:126]: Probably we should say somewhere that negative value decrements

**Review by karesti — CHANGES_REQUESTED** [review]

File: src/app/assets/languages/en.json (hunk 3 above)
Comment (dpanshug) [reply src/app/assets/languages/en.json:126]: Should we call this "Set delta" as it was mentioned in [marvel](https://marvelapp.com/prototype/8d66729/screen/69367851)
@dvagnero wdyt?

File: src/app/assets/languages/en.json [hunk 4]
```diff
@@ -121,7 +121,13 @@
...
+      "modal-delta-title": "Add delta to counter",
+      "delta-text-input": "Delta",
```
Comment (dvagnero) [inline src/app/assets/languages/en.json:128]: "Increment or decrement counters with arbitrary values. To subtract counter value use negative numbers." or something in those words.

**Review by dvagnero — COMMENTED** [review]
I added few suggestions. If you agree I can push my commit

**Comment by dpanshug** [comment]
> I added few suggestions. If you agree I can push my commit

Yes, go ahead with commit, thanks :)

**Review by karesti — APPROVED** [review]

**Comment by dpanshug** [comment]
@dvagnero I accidentally forced-pushed my commit for unit test before pulling yours.
I again made the changes for the docs you suggested. Please confirm.
Sorry for trouble.

File: src/app/Counters/AddDeltaCounter.tsx (hunk 1 above)
Comment (dpanshug) [reply src/app/Counters/AddDeltaCounter.tsx:6]: Added one unit test, let me know if we need some more cases in that.

**Comment by dvagnero** [comment]
@dpanshug No worries. The changes look good

**Review by dvagnero — APPROVED** [review]

**Pull request merged**